### PR TITLE
[MIG][9.0] account multicurrency revaluation

### PR DESCRIPTION
**Pull request by grindtildeath**

### CODE REVIEW

**Comment by fclementic2c** [comment]
@grindtildeath tell when travis is green so I can test on runbot first

**Comment by yvaucher** [comment]
@fclementic2c Have you tested it ?

**Comment by grindtildeath** [comment]
Ping @fclementic2c @elicoidal

**Review by elicoidal — APPROVED** [review]
LGTM

File: account_multicurrency_revaluation/README.rst [hunk 1]
```diff
@@ -21,7 +24,7 @@ To install this module, you need to:
 Configuration
 =============
 
-Due to the various legislation according the country, in the Company settings
+Due to the various legislation according the country, in the Accouting settings
```
Comment (elicoidal) [inline account_multicurrency_revaluation/README.rst]: Accounting

**Comment by fclementic2c** [comment]
test ok
:+1:

**Pull request merged**